### PR TITLE
TELCODOCS-146 - document nosmt to disable hyperthreading - release note

### PR DESCRIPTION
**Pull request by aireilly**
Release note for https://issues.redhat.com/browse/TELCODOCS-146

### CODE REVIEW

**Comment by aireilly** [comment]
Release note for telcodocs-146  - document `nosmt` to disable hyperthreading @MarSik @gsr-shanks please @ anyone else that needs to review. thanks.

**Comment by aireilly** [comment]
enterprise-4.7

**Comment by aireilly** [comment]
@kquinn1204 @abhatt-rh

File: release_notes/ocp-4-7-release-notes.adoc [hunk 1]
```diff
@@ -704,6 +704,12 @@ registry.redhat.io/rhpam-7/rhpam-smartrouter-rhel8:7.8.0
 
 * Previously on some systems, the installer would communicate with Ironic before it was ready and fail. This is now prevented. (link:https://bugzilla.redhat.com/show_bug.cgi?id=1915295[*BZ#1915295*])
 
+*Scale*
+
+* The `nosmt` additional kernel argument which configures hyperthreading was previously undocumented for use with {product-title}. To disable hyperthreading, create a performance profile that is appropriate for your hardware and topology, and then set `nosmt` as an additional kernel argument.
++
+For more information, see xref:../scalability_and_performance/cnf-performance-addon-operator-for-low-latency-nodes.html#about_hyperthreading_for_low_latency_and_real_time_applications_cnf-master[About hyperthreading for low latency and real-time applications].
```
Comment (ahardin-rh) [inline release_notes/ocp-4-7-release-notes.adoc:711]: .html/.adoc

Comment (aireilly) [reply release_notes/ocp-4-7-release-notes.adoc:711]: ack . fill fix

**Review by jeana-redhat — APPROVED** [review]

**Pull request merged**